src/implementations/twig/components/tabs/tabs.html.twig

```diff
@@ -59,27 +60,38 @@
 {# Print the result #}
 
 {% if _items is not empty and _items is iterable %}
-<nav class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+<div class="{{ _css_class }}"{{ _extra_attributes|raw }}>
```

Comment:
not sure the reason why of this change, but i guess there is.. :)

Reply:
Was following markup from w3: https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html but I guess we can keep `<nav>`